gitpull.yml

```diff
@@ -0,0 +1,25 @@
+- name: Pulling git code to ansible slave node
+  hosts: anotherinstance18.04
+  remote_user: ubuntu
+  sudo: yes
```

Comment:
Should you use sudo here?

Reply:
In code (gitpull.yml), when doing git pull it creates problem, Error : error: cannot open .git/FETCH_HEAD: Permission denied", "stderr_lines": ["error: cannot open .git/FETCH_HEAD: Permission denied"], thats why used sudo: yes. 

We can also use become:yes instead.